examples/guestbook/guestbook_ns/frontend-controller.json

```diff
@@ -0,0 +1,46 @@
+{
+    "kind": "ReplicationController",
+    "apiVersion": "v1beta3",
+    "metadata": {
+        "name": "frontend-controller",
+        "labels": {
+            "name": "frontend",
+            "template": "frontend"
+        }
+    },
+    "spec": {
+        "replicas": 2,
+        "selector": {
+            "name": "frontend"
+        },
+        "template": {
+            "metadata": {
+                "name": "frontend-controller",
+                "labels": {
+                    "name": "frontend",
+                    "uses": "redisslave,redis-master",
+                    "template": "frontend"
+                }
+            },
+            "spec": {
+                "volumes": null,
+                "containers": [
+                    {
+                        "name": "php-redis",
+                        "image": "kubernetes/example-guestbook-php-redis",
+                        "ports": [
+                            {
+                                "containerPort": 80,
+                                "hostPort": 8000
```

Comment:
Please remove hostPort. Example instructions can show how to use the apiserver's proxy to access the server.